### DISCUSSION TITLE
/radar/queues endpoint unexpectedly returning 404

### DISCUSSION THREAD

**Discussion by 1npo** [weather-gov/api#756] [General]
Hello,

I get a 404 response whenever I make a GET request to the /radar/queue endpoint with ldm1, ldm2, ldm3, ldm4, tds, or rds as the host.

The test request on [weather.gov](http://weather.gov/) returns the same, see screenshot below.

Am I using the wrong values for `host`? Or is there an issue with the endpoint? Or has the endpoint been deprecated and the OpenAPI spec just not updated yet?

![image](https://github.com/user-attachments/assets/692beca7-c618-4ca9-9897-1185c4946562)

Thanks,
Nick

**Comment by sullynole** [comment]
A couple issues going on here:

- ldm1, ldm2, ldm3, and ldm4 are not valid
- rds and tds are valid, but there are too many records to handle, which results in the 503 error (ticket open). In this case, workaround is to filter with additional parameters such as https://api.weather.gov/radar/queues/rds?station=KLWX

**Reply by 1npo** [reply]
I wasn't aware of any record limit for responses -- is that limit documented somewhere in the API spec and maybe I just overlooked it?

Somewhat tangential -- Is there a brief article that defines what the TDS, RDS, and LDMs are and how they're related to each other?

I found a few bits of information from Google and made a few assumptions:

- TDS is a [THREDDs Data Server](https://www.unidata.ucar.edu/software/tds/)
- RDS is a [Remote Data Server](https://www.weather.gov/rev/remotedata) that's used to ingest data from private/civilian weather stations
- LDM is a [Local Data Manager](https://docs.unidata.ucar.edu/ldm/current/factsheet.html) that runs on machines inside NWS weather stations and radars

But lots of assumptions about how NWS uses all this software, and I'm not really clear on how they work together.

**Comment by sullynole** [comment]
Good questions. LDM is a popular software used in the met community for data sharing, including radar data.  The rds and tds endpoints are specific to radar dataflow and populated by the radar operations center.  TDS and RDS are their primary and backup dataflow servers.  It would be best to reach out them for further insight into the radar data.  https://www.weather.gov/nl2/ServerView

The limit issue is a known issue on the queues endpoint that is mentioned on the Updates tab https://www.weather.gov/documentation/services-web-api

**Reply by 1npo** [reply]
That ServerView graphic is extremely helpful and just what I was looking for, thanks!

If I have any other questions during my research, is there a certain email/inbox I should reach out to with my questions?
